.gitignore

```diff
@@ -22,3 +22,4 @@ _testmain.go
 *.exe
 *.test
 *.prof
+.vscode/
```

Comment:
I would prefer to use .gitignore only for project-specific files. If you use a tool that creates something in project directory - add it to `.git/info/exclude` instead of `.gitignore`.